exec-unittests

```diff
@@ -186,18 +181,30 @@ for _major in ${major_list}; do
                 (( count++ ))
             fi
         done > ${_para_jobs_file}
-        parallel --will-cite -k --lb -a ${_para_jobs_file} ${para_jobs_arg} para_run_legacy
-        if [[ ${?} -ne 0 ]]; then
-            rc=1
+        if [[ -s ${_para_jobs_file} ]]; then
```

Comment:
I _think_ the code at line 189 which checks for a zero count is now pointless, because `count` will only be zero if `_para_jobs_file` is empty, in which case we won't enter the block.

If we don't enter the block here, then we know that `count` would be zero, so we don't actually need to calculate it (so we can remove the `let` statement and the increment at lines 177 and 181, respectively), and we can move the `if` at line 189 to an `elif` to this `if`.  (Of course, we should remove the `count` test from its condition, and we presumably need to add a check for not `"srcbld"`.)

Reply:
Fixed.

This pointed out other bad behaviors, though.  I made a number of updates so that we get the expected driver output, and corrected the list of unit tests sub-systems enumerated.